driver/src/models/state.rs

```diff
@@ -56,7 +57,7 @@ impl RollingHashable for State {
       let result = hasher.result();
       hash = result.to_vec();
     }
-    H256::from(hash.as_slice())
+    H256::from(from_slice(&hash))
```

Comment:
Why did this have to change?

Reply:
H256 is part of the new web3 library. And in this library, they enforce the from-operator to act on a [u8, 32] instead of &[u8].
I think they enforce it because it is safer.

Reply:
Does `H256::from_slice(hash.as_slice())` work?

Reply:
works... nice! I updated it